### DISCUSSION TITLE
I'am facing this problem when trying to run my hardhat.config.js

### DISCUSSION THREAD

**Discussion by mohamedtb20** [NomicFoundation/hardhat#2617] [Troubleshooting] [answered]
C:\Users\DELL\nft_marketplace>npx hardhat test
Error HH8: There's one or more errors in your config file:

  * Invalid account: #0 for network: mumbai - Expected string, received function
  * Invalid account: #0 for network: mainnet - Expected string, received function

To learn more about Hardhat's configuration, please go to https://hardhat.org/config/

For more info go to https://hardhat.org/HH8 or run Hardhat with --show-stack-traces

**Comment by alcuadrado** [comment, marked as answer]
Duplicate of https://github.com/NomicFoundation/hardhat/discussions/2615
